### PR TITLE
Added confirm input  in profile settings

### PR DESCRIPTION
**Pull request by Syed-Ansar**
Added Confirm password field for profile-change password section and Validated inputs as if there is no input in current passwords and having inputs in other to new passwords then error will be thrown and if confirm password doesn't match with new password then it with throw an error. Recreated pull request because of linting problems

fixes issue https://github.com/ToolJet/ToolJet/issues/3108

tooljet

### CODE REVIEW

File: frontend/src/SettingsPage/SettingsPage.jsx [hunk 1]
```diff
@@ -45,22 +47,41 @@ function SettingsPage(props) {
...
+          setCurrentPassword('');
+          setNewPassword('');
+          setConfirmPassword('');
+        } catch (error) {
+          toast.error('Please verify that you have entered the correct password', {
+            duration: 3000,
+          });
+        }
+      } else {
+        toast.error('Please verify that confirm new password is incorrect', {
```
Comment (gsmithun4) [inline frontend/src/SettingsPage/SettingsPage.jsx:66]:
```suggestion
        toast.error('New password and confirm password are not matching', {
```

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

**Comment by Syed-Ansar** [comment]
Hi @gsmithun4 I have made changes as you suggested.

**Comment by gsmithun4** [comment]
Deployed
https://tj-3375.herokuapp.com/

**Comment by Nandinisaha13** [comment]
Screenshot (71)
Screenshot (70)
There is some extra white space at the bottom for both light & dark mode. @Syed-Ansar can you look into this?

**Comment by Syed-Ansar** [comment]
Hey @Nandinisaha13 I have just taken screen-shots on deployed [Here](https://tj-3375.herokuapp.com/). I did'nt get any whitespace 

Screenshot 2022-06-27 at 12 18 53 PM

Screenshot 2022-06-27 at 12 18 46 PM

**Comment by Nandinisaha13** [comment]
@Syed-Ansar That is being checked because I am checking the same and I can see the white space still, may be a resolution problem! I also found this(screenshot attached) If we can give a error message as "Current password is incorrect" or "Please verify the password entered in current password or may be something more suitable.
Screenshot (182)

**Comment by Syed-Ansar** [comment]
> @Syed-Ansar That is being checked because I am checking the same and I can see the white space still, may be a resolution problem! I also found this(screenshot attached) If we can give a error message as "Current password is incorrect" or "Please verify the password entered in current password or may be something more suitable. Screenshot (182)

Hey @Nandinisaha13 I am using Macbook Air so I am not able to detect the problem of white-space, I verified on Chrome as well as Firefox I couldn't detect the whitespace problem on either themes i.e, Light & Dark. When it comes to password toast message it was set before only so I didn't changed it. If you ask me to change I would suggest as "You have entered an invalid password" or "You have entered an invalid current password".

**Comment by gsmithun4** [comment]
> > @Syed-Ansar That is being checked because I am checking the same and I can see the white space still, may be a resolution problem! I also found this(screenshot attached) If we can give a error message as "Current password is incorrect" or "Please verify the password entered in current password or may be something more suitable. Screenshot (182)
> 
> Hey @Nandinisaha13 I am using Macbook Air so I am not able to detect the problem of white-space, I verified on Chrome as well as Firefox I couldn't detect the whitespace problem on either themes i.e, Light & Dark. When it comes to password toast message it was set before only so I didn't changed it. If you ask me to change I would suggest as "You have entered an invalid password" or "You have entered an invalid current password".

@Nandinisaha13 Let's create separate issues for the cases, Good to merge if functionality works

**Comment by Nandinisaha13** [comment]
<html>
<body>
<google-sheets-html-origin>Test Cases

Test Case | Steps | Result
-- | -- | --
Should be able to view confirm new password field. | Hover over name annotation. | 2 options is displayed.
  | Go to profile. | Profile page is displayed.
  | Observe. | Confirm new password field is displayed.
Should be able to confirm new password. | Hover over name annotation. | 2 options is displayed.
  | Go to profile. | Profile page is displayed.
  | Enter correct current password. | Current password is displayed in dots.
  | Enter new password. | New password is displayed in dots.
  | Enter confirm new password same as new password. | Confirm new password field is displayed in dots.
  | Click on change password. | The password is changed.
Should not be able to confirm new password. | Hover over name annotation. | 2 options is displayed.
  | Go to profile. | Profile page is displayed.
  | Enter correct current password. | Current password is displayed in dots.
  | Enter new password. | New password is displayed in dots.
  | Enter confirm new password not same as new password. | Confirm new password field is displayed in dots.
  | Click on change password. | Error is displayed.


</body>
</html>

**Review by gsmithun4 — DISMISSED** [review]
LGTM

**Review by gsmithun4 — APPROVED** [review]
LGTM

**Pull request merged**